.coveragerc

```diff
@@ -6,9 +6,11 @@ source =
     tests/unit
 omit =
     lms/scripts/*
+    lms/extensions/feature_flags/views/test.py
+    lms/views/feature_flags_test.py
```

Comment:
Why are these files not subject to coverage? A comment here would be nice, otherwise it just lives in heads and is hard to question.

Reply:
They're just weird little test views for the feature flags mechanism. They're not anything actually being used by users. They could even just be deleted but they can be handy now and then

Reply:
Added a comment